webpack/assets/javascripts/react_app/components/ChartBox/ChartBoxSelectors.js

```diff
@@ -0,0 +1,11 @@
+import {
+  selectAPIResponse,
+  selectAPIError,
+  selectAPIStatus,
+} from '../../redux/API/APISelectors';
+
+export const selectChart = (state, key) => ({
+  ...selectAPIResponse(state, key),
+  error: selectAPIError(state, key),
+  status: selectAPIStatus(state, key),
+});
```

Comment:
can we split it into multiple selectors?

Reply:
Done